core/textile/sync/restore.go

```diff
@@ -0,0 +1,40 @@
+package sync
+
+import (
+	"context"
+
+	"github.com/FleekHQ/space-daemon/log"
+)
+
+// replicate a local thread on the hub
+func (s *synchronizer) restoreBucket(ctx context.Context, bucket string) error {
+
+	b, err := s.getBucket(ctx, bucket)
```

Comment:
Should this default to default bucket? Just wondering because if someone calls this on any bucket but `personal`, it will restore files from mirror path into that bucket :thinking:

Reply:
@jsonsivar 

I think it's a broader problem of allowing harmful operations...